### DISCUSSION TITLE
Upgrade package in multiple workspaces

### DISCUSSION THREAD

**Discussion by DanielBailey-web** [vercel/turborepo#957] [Help] [answered]
Is there a good (fast/easy) way to upgrade a package used in all my workspaces (e.g. tailwindcss) or should I just do it workspace by workspace?

**Comment by marek-hanzal** [comment]
For example when you're using `pnpm`, you can do `pnpm up --latest -r` when `-r` is recursive (that important parameter); this one goes through all and updates everything, making changes also in all `package.json`s.

**Comment by anthonyshew** [comment, marked as answer]
We (finally) have some documentation around this: https://turbo.build/repo/docs/crafting-your-repository/managing-dependencies#keeping-dependencies-on-the-same-version

Happy to hear your feedback if things are still unclear.
